packages/react-server-cli/src/compileClient.js

```diff
@@ -0,0 +1,222 @@
+import webpack from "webpack"
+import path from "path"
+import mkdirp from "mkdirp"
+import fs from "fs"
+import ExtractTextPlugin from "extract-text-webpack-plugin"
+
+// commented out to please eslint, but re-add if logging is needed in this file.
```

Comment:
What's the project feeling about suppressing linting rules? I sort of assumed that it would be frowned upon unless completely necessary.